pages/pipelines/conditionals.md

```diff
@@ -93,6 +93,9 @@ notify:
 ```
 {: codeblock-file="pipeline.yml"}
 
+>📘 Evaluating conditionals based on build state
+> Conditional expressions on build state will be available only at pipeline level and should not be used at step level.
```

Comment:
I think using the callout draws too much attention to this point. It can probably be included as regular text:

```suggestion
Note that conditional expressions on the build state are only available at the pipeline level. You can't use them at the step level.
```

What do you think?